bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php

```diff
@@ -1662,4 +1671,32 @@ public function suggestClassIdentifierAction()
 
         return $this->adminJson($result);
     }
+
+
+    /**
+     * @Route("/suggest-custom-layout-identifier")
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function suggestCustomLayoutIdentifierAction(Request $request)
+    {
+        $classId = $request->get('classId');
+
+        $db = Db::get();
```

Comment:
shouldn't those sql queries sit inside the dao?

Reply:
done :wink:

Reply:
perfect 👍